meta-lmp-base/recipes-core/os-release/os-release.bbappend

```diff
@@ -9,3 +9,9 @@ SUPPORT_URL = "https://support.foundries.io/"
 LMP_MACHINE = "${MACHINE}"
 LMP_FACTORY = "${LMP_DEVICE_FACTORY}"
 LMP_FACTORY_TAG = "${LMP_DEVICE_REGISTER_TAG}"
+
+do_install_append () {
```

Comment:
Here we should use do_deploy instead, as the main idea is to deploy the same file that gets installed (which is the main task used for building the final recipe-based package).

Reply:
`do_deploy` or `do_deploy_append` since we are inheriting the original recipe?

Reply:
There is no do_deploy on the original recipe, so either would work.

But to be safe I think doing append is better, in case later on the original recipe starts having its own function.